build-system/server.js

```diff
@@ -561,6 +561,22 @@ app.get('/dist/rtv/99*/*.js', function(req, res, next) {
   }).catch(next);
 });
 
+app.get('/dist/rtv/*/v0/*.js', function(req, res, next) {
+  var mode = getPathMode(req.headers.referer);
+  if (process.argv.includes('test', 2)) {
```

Comment:
we can rename `getPathMode` to `getMode`, so it should just take care all the logic, if that's what you worried about?